plugins/strict-connection-checker/package.json

```diff
@@ -44,7 +43,7 @@
   "devDependencies": {
     "@blockly/dev-scripts": "^1.1.0",
     "@blockly/dev-tools": "^2.0.1",
-    "blockly": "git://github.com/google/blockly.git#develop",
+    "blockly": "^3.20200924.1",
     "chai": "^4.2.0"
   },
   "peerDependencies": {
```

Comment:
Since this won't work on earlier versions of Blockly. You'll want to add a min cap on peer dependencies as well. eg: ">3.20200924.1"

Reply:
Done.